### DISCUSSION TITLE
libprotobuf linker error

### DISCUSSION THREAD

**Discussion by yunusyurtturk** [pelicanmapping/osgearth#2587] [Support Q&A] [answered]
When building osgEarth with vcpkg option, during build process (Visual Studio > osgEarth Project > Build)
It gives an error, complaining about libprotobuf definitions not found.

<img width="628" alt="image" src="https://github.com/user-attachments/assets/7cba2343-66b5-4558-ad8b-a99a57fa3ef7">

I asked ChatGPT and its first response was 'change it to .lib'.
I tried by changing it to 

..\..\..\lib\RelWithDebInfo\osgEarth.lib

Now it doesn't complain anymore. 
Does that mean I made an error while setting up the project, or there was already a problem about libprotobuf?

**Comment by gwaldron** [comment]
Weird. I have not seen this issue - are you using the latest 3.6?

**Reply by yunusyurtturk** [reply]
I've tried 3.6, 3.4 previously and currently trying 3.5.
With manual workarounds I mentioned [here](https://github.com/gwaldron/osgearth/discussions/2589) I'm able build & run at least some of the projects with version 3.5. 

I feel more experienced about overal build & run process now so I will try one more time with version 3.6

**Reply by yunusyurtturk** [reply, marked as answer]
It is fine with version 3.6
